frontend/src/app/module-blueprint/components/comment-section/comment-section.component.css

```
@@ -29,108 +61,101 @@
 .comment-date {
-  font-size: 0.8rem;
-  opacity: 0.6;
+  font-size: 11.5px;
+  color: var(--bni-faint);
 }
 
 .comment-body {
-  margin: 0.15rem 0;
+  margin: 3px 0 6px;
   white-space: pre-wrap;
   word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)